domains/Kapilgiree.json

```
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Kapilgiree",
```

```suggestion
        "username": "Yoooo-cyber",
```

username should match your github username